<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCollectionForm</class>
 <widget class="QDialog" name="AddCollectionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>561</height>
    </rect>
   </property>
   <property name="title">
    <string>Add New Collection</string>
   </property>
   <widget class="QLabel" name="cid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Collection ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="datatype">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="category">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Version</string>
    </property>
   </widget>
   <widget class="QLabel" name="publisher">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Publisher</string>
    </property>
   </widget>
   <widget class="QLabel" name="year">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Record Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="duration">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeCBB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="datatypeCBB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="cateGroup">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="animationCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Animation</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="adventureCB">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adventure</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="fantasyCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fantasy</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dramaCB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Drama</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="comedyCB">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Comedy</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="yearCBB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="cidErr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleErr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="versionErr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="publisherErr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="durationErr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>500</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="cidF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>19</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="versionF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="publisherF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="durationF">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="numcopiesCBB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="year_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Num Copies</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
